tests/test_base_loader_integration.py

```diff
@@ -0,0 +1,86 @@
+import ast
+import copy
+import json
+import logging
+import os
+import sys
+import uuid
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from loader import base_loader
+from tests.abstract_loader_test import AbstractLoaderTest
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+
+class TestBaseLoaderIntegration(AbstractLoaderTest):
+    """Integration tests for base_loader.py."""
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.google_project_id = 'platform-dev-178517'
+
+        # Service account: travis-underpriveleged-tester@platform-dev-178517.iam.gserviceaccount.com
+        # Has only viewer level permissions, while the bucket requires at least editor level.
+        # Only use these permissions for the tests in this file.
+        underprivileged_credentials = os.path.abspath('underprivileged_credentials.json')
```

Comment:
Since this is in a couple different places I'm wondering if it could be abstracted to the parent class. Maybe as a helper function that you can call here